build.xml

```diff
@@ -231,9 +231,8 @@
         </exec>
     </target>
 
-    <target name="dump-translations" depends="dump-translations-project-base, dump-translations-packages" description="Extracts translatable messages in all packages ."/>
-
-    <target name="dump-translations-packages" description="Extracts translatable messages in all packages .">
+    <target name="dump-translations" description="Extracts translatable messages in the whole monorepo.">
+        <phingcall target="shopsys_framework.dump-translations"/>
```

Comment:
You're right about that (and I even explain why in the contribution guidelines). The target has no dependencies so there are no worries about that. Also, I'd rather not mix dependencies and tasks, as it might confuse people...